rfcs/rfc-filing-technical-corrections.md

```diff
@@ -0,0 +1,78 @@
+- Start Date: 2021-02-25
+- Target Major Version: (EPIC or User Story TAG|Link)
+- Reference Issues: n/a
+- Entity Issue: (leave this empty)
+- Implementation PR: (leave this empty)
+
+# Summary
+
+An implementation to support technical changes to a customer filing done directly by support engineers.
+
+# Basic example
+
+A failure in the validation rules may have allowed for an address like this
+``` 
+'deliveryAddress': {
+    'streetAddress': '1 Main st.',
+    'addressCity': 'Vancouver',
+    'addressCountry': 'CA',
+    'postalCode': 'H0H0H0',
+    'addressRegion': 'B'
+},
+```
+where it can clearly be deduced that the Region should be **BC** and not just **B**.
+The submitted JSON can be corrected, and still applied to the business model correctly.
+``` 
+'deliveryAddress': {
+    'streetAddress': '1 Main st.',
+    'addressCity': 'Vancouver',
+    'addressCountry': 'CA',
+    'postalCode': 'H0H0H0',
+    'addressRegion': 'BC'
+},
+```
+
+
+# Motivation
+
+There are times when a filing gets accepted, but due to an error in the validation it may have technical issues that make applying the filing impossible.
+In these cases it has already been paid for and because it cannot be applied, it cannot even be corrected by customer our business staff.
+
+The filing_json as submitted by the customer must be treated as immutable, so that it can always be retrieved and used in cases of legal challenges.
+
+Where the error is simple to understand, a corrected version can be used to get the filing to apply properly.
+Further changes can be filed as corrections by business staff.
+
+
+# Detailed design
+
+Adding a tech-json attribute to the Filing model. If this field is populated it will get used by the Filer and reporting services (until marked complete).
+A note on the business should also be created with the fix and the tech staff id that implemented the fix.
```

Comment:
Who would create this note (is it visible to the public ie a detail comment on the ledger?

Reply:
That's a good idea, especially as any notes are FOI'able you'd want it to be as correct and informative as possible.